images/deployer/Dockerfile

```diff
@@ -9,4 +9,11 @@
 #
 FROM openshift/origin
 
+RUN yum install -y nfs-utils sudo
+
+# Edit sudoers file
+# To avoid error: sudo: sorry, you must have a tty to run sudo
+RUN sed -i -e "s/Defaults    requiretty.*/ #Defaults    requiretty/g" /etc/sudoers
```

Comment:
I don't know what the implications of adding `sudo` to the deployer image are... how does that expand abilities for normal deployer pods?